src/assets/scripts/client/aircraft/FlightManagementSystem/Fms.js

```diff
@@ -699,11 +703,6 @@ export default class Fms {
      * @param runwayModel {RunwayModel}
      */
     replaceDepartureProcedure(routeString, runwayModel) {
-        // this is the same procedure that is already set, no need to continue
-        if (this.hasLegWithRouteString(routeString)) {
-            return;
-        }
-
         if (this.departureRunwayModel.name !== runwayModel.name) {
             this.setDepartureRunway(runwayModel);
```

Comment:
Not really. I couldn't think of a good way to get it done so I just left that bit as it was. We need to be able to:
- assign a new runway and have the Sid leg regenerated
- assign a new SID and have some way to prevent aircraft from attempting to use it from their current expected runway when it's invalid for that procedure
    - we could forcibly reassign a new runway, though how would we deal with aircraft already in the queue when they get the new Sid?
    - we could simply reject the Sid to the user with a red message, so they will have to change their departure runway before being able to accept the reroute (problematic because currently no `expect` command)